api/oc_ri_server.c

```
@@ -17,19 +17,249 @@
+OC_MEMB(g_app_resources_s, oc_resource_t, OC_MAX_APP_RESOURCES);
+
 OC_LIST(g_on_delete_resource_cb_list);
 OC_MEMB(g_on_delete_resource_cb_s, oc_ri_on_delete_resource_t,
         OC_MAX_ON_DELETE_RESOURCE_CBS);
 
+oc_resource_t *
+oc_ri_alloc_resource(void)
+{
+  return oc_memb_alloc(&g_app_resources_s);
+}
+
```

cppcheck

[error] There is an unknown macro here somewhere. Configuration is required. If oc_ri_on_delete_resource_find_callback is a macro then please configure it.

(unknownMacro)